cmd/minishift/cmd/start.go

```diff
@@ -187,13 +189,21 @@ func runStart(cmd *cobra.Command, args []string) {
 		atexit.Exit(1)
 	}
 
+	automountHostfolders(host.Driver)
```

Comment:
I had it in provisioner first, but that only runs once ;-), so this is the only place for now...